Solution_Matrix.java

```diff
@@ -0,0 +1,49 @@
+/*
+Time Complexity :  O(n), here I am confussed like I am checking all elements only for row 
+how to count time complexity of this could you please guid me on this ?
+
+Space:  O(1) // as we are not using any auxillary space.

```

Comment:
Here you are going across all the elements in the matrix in the worse case. You can use the given constraint that the matrix is sorted and use Binary Search where the Time Complexity will be `O(logn)`